README.md

```diff
@@ -0,0 +1,218 @@
+o-layout [![Circle CI](https://circleci.com/gh/Financial-Times/o-layout/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-layout/tree/master)
+=================
+
+This component provides a responsive full page layout for internal documentation and tooling.
+It provides:
+- consistent design (typography, colors),
+- ease of set up (header, footer, sidebar, navigation)
+
+
+## Table of Contents
+
+- [Usage](#usage)
+	- [Markup](#markup)
+		- [Layout Base](#layout-base)
+		- [Navigation and Content](#navigation-and-content)
+		- [Asides](#asides)
+		- [Tables](#tables)
+	- [Sass](#sass)
+	- [JavaScript](#javascript)
+		- [Construction](#construction)
+		- [Custom Navigation](#custom-navigation)
+- [Contact](#contact)
+- [Licence](#licence)
+
+
+## Usage
+
+`o-layout` provides a grid that has the following structure:
+
+```
+┌————————————————————————————┐
+|           HEADER           |
+├————————————————————————————┤
+|        |                   |
+|  SIDE  |    MAIN           |
+|  BAR   |    CONTENT        |
+|        |                   |
+├————————————————————————————┤
+|           FOOTER           |
+└————————————————————————————┘
+```
+
+Within the main content section, there is another grid, which looks like this:
+
+```
+┌————————————————————————————┐
+|                            |
+├————————————————————————————┤
+|        | MAIN       |      |
+|        | CONTENT    |      |
+|        | WELL       |      |
+|        |            |      |
+├————————————————————————————┤
+|                            |
+└————————————————————————————┘
+```
+At is base, the main content section styles _elements_, not classes.
+It will automatically style paragraphs, headings, lists and anchor tags, for example.
+It will also automatically style tables and asides, which will occupy different columns or span different columns within the main content section.
+
+The grid does not require the sidebar to maintain its layout.
+
+### Markup
+
+`o-layout` uses CSS Grid Layout, and more specifically [grid template areas](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Grid_Template_Areas).   
+
+#### Layout Base
+The markup below ↓ will generate the ascii grid above ↑.  
+
+The main content section will constrain is _immediate_ children to its first column, with the exception of `table`s and `asides`.  
+If you want an element to span the full width of the main content area, you can apply the `o-layout__main--full-span` class to that element to achieve that effect.
```

Comment:
No extra empty line between these two sentences means that a `<br/>` is inserted, is that intentional?